### PR TITLE
updates html pages

### PR DESCRIPTION
**Pull request by andrewjtech123**
I realized that we probably need to regen the docs to get those changes published from the linked PR. I've never regened docs in this repo before though so not sure if this is right. 

https://github.com/Shopify/js-buy-sdk/pull/723/files

### CODE REVIEW

File: docs/collection-resource.js.html [hunk 1]
```diff
@@ -106,9 +106,9 @@ <h1 class="page-title">Source: collection-resource.js</h1>
    * @param {String} id The id of the collection to fetch.
    * @return {Promise|GraphModel} A promise resolving with a `GraphModel` of the collection.
    */
-  fetchWithProducts(id) {
+  fetchWithProducts(id, {productsFirst = 20} = {}) {
```
Comment (DrTeePot) [inline docs/collection-resource.js.html]: First relevant change, LGTM, although maybe someone should have updated the comment too when this change was made.

File: docs/index.html [hunk 2]
```diff
@@ -135,23 +160,30 @@ <h3 id="initializing-the-client">Initializing the Client</h3>
 
 // Fetch a single collection by ID, including its products
 const collectionId = 'Z2lkOi8vc2hvcGlmeS9Db2xsZWN0aW9uLzM2OTMxMjU4NA==';
+// Set a parameter for first x products, defaults to 20 if you don't provide a param
 
-client.collection.fetchWithProducts(collectionId).then((collection) => {
+client.collection.fetchWithProducts(collectionId, {productsFirst: 10}).then((collection) => {
```
Comment (DrTeePot) [inline docs/index.html]: Relevant change

File: docs/index.html [hunk 3]
```diff
@@ -225,16 +269,22 @@ <h3 id="initializing-the-client">Initializing the Client</h3>
 // Update the shipping address for an existing checkout.
 client.checkout.updateShippingAddress(checkoutId, shippingAddress).then(checkout => {
   // Do something with the updated checkout
-});</code></pre><h2 id="expanding-the-sdk">Expanding the SDK</h2>
+});
+</code></pre>
+<h3 id="completing-a-checkout">Completing a checkout</h3>
```
Comment (DrTeePot) [inline docs/index.html]: Completing a checkout! This LGTM

File: docs/collection-resource.js.html (hunk 1 above)
Comment (StefanSlehta) [reply docs/collection-resource.js.html]: That's me, I totally forgot to update the comment

Comment (DrTeePot) [reply docs/collection-resource.js.html]: That's fine! Could you do that when you have a moment and we'll update the documentation again? I don't think that's blocking for this PR.

**Comment by DrTeePot** [comment]
Looks like these went out in https://github.com/Shopify/js-buy-sdk/pull/725/files#diff-eae600270579316beb8b8d9b9a5732d4R230

**Comment by DrTeePot** [comment]
Thanks for this PR @andrewjtech123 , looks like the docs got regenerated when @rebeccajfriedman released a new version.

**Pull request closed without merging**